inc/namespace.php

```diff
@@ -75,4 +75,32 @@ function bootstrap() {
 		define( 'ALTIS_ANALYTICS_PINPOINT_ENDPOINT', getenv( 'ALTIS_ANALYTICS_PINPOINT_ENDPOINT' ) );
 		define( 'ALTIS_ANALYTICS_COGNITO_ENDPOINT', getenv( 'ALTIS_ANALYTICS_COGNITO_ENDPOINT' ) );
 	}
+
+	add_filter( 'qm/output/file_path_map', __NAMESPACE__ . '\\set_file_path_map', 1 );
+}
+
+/**
+ * Enables Query Monitor to map paths to their
+ * original values on the host.
+ *
+ * @param array $map an associative array of folders as keys, and their mappings as values
+ * @return array the adjusted mapping of folders
+ */
+function set_file_path_map( array $map ) : array {
+	if ( ! file_exists( '/etc/chassis-constants' ) ) {
+		return $map;
+	}
+	$json_string = file_get_contents( '/etc/chassis-constants' );
+	if ( empty( $json_string ) ) {
+		return $map;
+	}
+	$data = json_decode( $json_string, true );
+	if ( empty( $data['synced_folders'] ) ) {
```

Comment:
I'd prefer an explicit `empty( $data )` here, as it's a different code path.
```suggestion
	if ( empty( $data ) || empty( $data['synced_folders'] ) ) {
```